angular/src/app/project-form/project-form.component.ts

```diff
@@ -35,7 +30,7 @@ export class ProjectFormComponent implements OnInit {
         console.log(this.projectForm.value);
       });
     } else {
-      this.projectFormService.edit(this.projectForm.value, this.project._id).subscribe((response) => {
+      this.projectFormService.edit(this.projectForm.value, this.project.id).subscribe((response) => {
         this.response = response;
         console.log(response);
```

Comment:
Here

Reply:
Sorry, what needs to be changed here?

Reply:
Ah I was just refering again to the console log statement

Reply:
I think most of our components have these statements - I didn’t add any of these for example. They’re useful to make sure the request went through when debugging, not sure they’re necessary to take out and if we’re going to take them out, we should do so everywhere not just here.

Reply:
Yeah we should remove the other console log statement as well, if you could remove the ones involved in this PR I will go ahead and clean the rest. They could be helpful but print debugging is still part of the debugging process, which should be kept in local branches if possible. I don't mean to pick on you on this specifically, I just started to focus on keeping things cleaner :P